lib/kitchen/driver/ssh_base.rb

```diff
@@ -115,6 +115,14 @@ def verify(state) # rubocop:disable Metrics/AbcSize
       rescue Kitchen::Transport::TransportFailed => ex
         raise ActionFailed, ex.message
       ensure
+        if verifier[:scp_file]
```

Comment:
I still don't think this needs to be in an `ensure` block.  If your serverspec command fails, does it raise an exception?  If so, which one?

There are already 4 commands being run on the remote host in the `begin`/`rescue` block.  If any of those 4 fail except the serverspec command then the `download` command below will also fail.  That could mask a real failure by raising, instead, a failure from the `download` command.

Does my concern make sense?